### PR TITLE
Object manipulation 3

### PR DESCRIPTION
**Pull request by jacobhusband**
Screen Shot 2022-09-02 at 4 53 52 PM

### CODE REVIEW

File: object-manipulation-3/main.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+// Deal cards
+playerArray[0].hand.push(shuffledDeck[0]);
+playerArray[0].hand.push(shuffledDeck[1]);
+playerArray[1].hand.push(shuffledDeck[2]);
+playerArray[1].hand.push(shuffledDeck[3]);
+playerArray[2].hand.push(shuffledDeck[4]);
+playerArray[2].hand.push(shuffledDeck[5]);
+playerArray[3].hand.push(shuffledDeck[6]);
+playerArray[3].hand.push(shuffledDeck[7]);
```
Comment (codingcodymiller) [inline object-manipulation-3/main.js:71]: You should make this into a loop

File: object-manipulation-3/main.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      cardDeck.push({ rank, suit: 'clubs' });
+    } else if (l === 1) {
+      cardDeck.push({ rank, suit: 'diamonds' });
+    } else if (l === 2) {
+      cardDeck.push({ rank, suit: 'hearts' });
+    } else if (l === 3) {
+      cardDeck.push({ rank, suit: 'spades' });
+    }
+  }
+}
```
Comment (codingcodymiller) [inline object-manipulation-3/main.js:49]: I think you could simplify this significantly by making an array of possible ranks and an array of possible suits and looping over those to generate each card.

File: object-manipulation-3/main.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+console.log(
+  'Players and their scores: ',
+  playerArray[0],
+  playerArray[1],
+  playerArray[2],
+  playerArray[3]
+);
+
+console.log('Winning player: ', playerArray[maxIndex]);
```
Comment (codingcodymiller) [inline object-manipulation-3/main.js:110]: You should try to account for ties as well 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by jacobhusband** [comment]
Accounting for ties now: 
Screen Shot 2022-09-07 at 12 08 59 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**